release-notes/10.0/preview/preview1/winforms.md

```diff
@@ -1,14 +1,71 @@
 # Windows Forms in .NET 10 Preview 1 - Release Notes
 
-.NET 10 Preview 1 includes new Windows Forms features & enhancements:
+WinForms continues to ease the pain of migrating away from BinaryFormatter in .NET 10 Preview 1 by simplifying Clipboard serialization for certain custom types. In .NET 9, calls to BinaryFormatter to perform serialization were updated to throw `PlatformNotSupportedException` if the developer did not reference an unsupported compatibility package and enable BinaryFormatter via an application configuration switch([see more](https://learn.microsoft.com/dotnet/standard/serialization/binaryformatter-migration-guide/compatibility-package)). Clipboard and Drag/Drop scenarios which use custom types, as of .NET 9, require the developer to either enable BinaryFormatter or explicitly take action to use JSON serialization (or similar) to succeed. In .NET 10 Preview 1, WinForms is introducing APIs to help simplify the process of JSON serialization in Clipboard and Drag/Drop scenarios and obsoleting existing APIs to continue to encourage developers to move away from the unsafe BinaryFormatter serialization. For cases where BinaryFormatter is still required, a configuration switch has been added.
 
-- [Feature](#feature)
 
-Windows Forms updates in .NET 10:
+For in-depth discussion of the APIs involved, and usage examples, please see the original [API Proposal](https://github.com/dotnet/winforms/issues/12362) in the WinForms repo.
+
+### Obsoleted APIs
+
+In .NET 10, WinForms has marked as obsolete the following APIs to help warn developers that BinaryFormatter may be used to deserialize data. For detailed usage information, please see the API proposal located *An external link was removed to protect your privacy.*.
+
+**Clipboard:**
+
+- `public object GetData(string format)`
+
+**DataObject:**
+
+- `public virtual object? GetData(string format)`
+- `public virtual object? GetData(string format, bool autoConvert)`
+- `public virtual object? GetData(Type format)`
+
+**ClipboardProxy (VisualBasic wrapper for WinForms Clipboard):**
+
+- `public object GetData(string format)`
+
+### New APIs
+
+We’ve introduced the following APIs:
+
+**Clipboard:**
 
-- [What's new in Windows Forms in .NET 10](https://learn.microsoft.com/dotnet/desktop/winforms/whats-new/net100) documentation
+- `public void SetDataAsJson<T>(T data) { }`
+- `public void SetDataAsJson<T>(string format, T data) { }`
+- `public void SetDataAsJson<T>(string format, bool autoConvert, T data) { }`
+- `public virtual bool TryGetData<T>(out T data) { }`
+- `public virtual bool TryGetData<T>(string format, out T data) { }`
+- `public virtual bool TryGetData<T>(string format, bool autoConvert, out T data) { }`
+- `public virtual bool TryGetData<T>(string format, Func<Reflection.Metadata.TypeName, Type> resolver, bool autoConvert, out T data) { }`
 
+**IDataObject:**

```

Comment:
```suggestion
**ITypedDataObject:**

```

Reply:
@merriemcgaw  - see Jeremy Barton's recommendations summary he added to the API proposal bug after the meeting - he made some modifications to the original proposal